internal/ui/keyboard_js.go

```diff
@@ -0,0 +1,26 @@
+// +build js
+
+package ui
+
+import (
+	"unicode"
+
+	"github.com/gopherjs/gopherjs/js"
+)
+
+func keypress(e *js.Object) {
+	if runebuffer != nil {
+		if r := rune(e.Get("charCode").Int()); unicode.IsPrint(r) {
+			runebuffer = append(runebuffer, r)
+		}
+	}
+}
+
+func Keyboard() []rune {
```

Comment:
So Keyboard has side effect to clear the buffer? This is a different behavior from Touches or other API. Is it possible to clear the buffer when a new frame comes instead of when Keyboard is called?